src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs

```diff
@@ -15,6 +16,12 @@ internal class ManagedProjectSystemPackage : AsyncPackage
         public const long EditProjectFileCmdId = 0x1001;
         public const long GenerateNuGetPackageProjectContextMenuCmdId = 0x2000;
         public const long GenerateNuGetPackageTopLevelBuildCmdId = 0x2001;
+        public const string DefaultCapabilities = ProjectCapability.AppDesigner + "; " +
+                                                  ProjectCapability.EditAndContinue + "; " +
+                                                  ProjectCapability.HandlesOwnReload + "; " +
+                                                  ProjectCapability.OpenProjectFile + "; " +
+                                                  ProjectCapabilities.ProjectConfigurationsInferredFromUsage + "; " +
+                                                  ProjectCapabilities.LanguageService;
```

Comment:
There shouldn't be any functional changes with this: if shared projects get these capabilities now, they would have gotten them before, as the `DefaultCapabiltiesProvider` was marked as `AlwaysApplicable`